base/Dockerfile

```diff
@@ -37,3 +37,7 @@ CMD ["run"]
 
 # LABEL must be last for proper base image discoverability
 LABEL repository.socrata/base=""
+
+# Add locale profiles and default to en_US.UTF-8
+RUN locale-gen en_US.UTF-8
```

Comment:
That sounds right (more language/environment agnostic). Confirming that it exists in 14.04.